webapp/src/features/home/Home.tsx

```diff
@@ -11,16 +11,20 @@ const helpfulLinks = [
         title: "S4 Drive Folder",
         text: "All the spreadsheets and documents relevant to hosting can be found here.",
         url: "https://drive.google.com/drive/u/1/folders/1LSc92ZMwD4B68Ocx0dwKJSE2Eek21SET",
-    },
-    {
-        title: "Performer Interest Sheet (Q2)",
-        text: "This is where the performers actually sign up.",
-        url: "https://docs.google.com/spreadsheets/d/1xzejwFYiaFkS7atfWf9YJe7nmZqF_UxwZBjkigoL4DI/edit?usp=sharing",
-    },
-    {
-        title: "Streaming Assets",
-        text: "Images and logos that are used in our streams, including DJ logos.",
-        url: "https://drive.google.com/drive/u/1/folders/1tuZddCNBcFpVFjyk3XKB6M1d64mAnjX8",
+        extra: [
+            {
+                title: "Signup Sheet",
+                url: "https://docs.google.com/spreadsheets/d/1ANiFOmz1_ADaMxCtcQE0HdGq3TsJM__Whg87DDN_y9Q/edit?usp=drive_link"
```

Comment:
Agh, the signup sheet was labeled something else in the drive.

Would you be able to swap this out with what used to be on the old "performer interest sheet" card?

https://docs.google.com/spreadsheets/d/1xzejwFYiaFkS7atfWf9YJe7nmZqF_UxwZBjkigoL4DI/edit?usp=sharing

Reply:
Sure!